### DISCUSSION TITLE
Hyperlink in MD file

### DISCUSSION THREAD

**Discussion by tlsLizard** [community/community#24030] [New to GitHub]
<p>Hello World!</p>
<p>I would like to add a link to my MD file of :<br>
tlsLizard/collector</p>
<p>I’m using this syntax : ```<br>
<a>text</a> : text in brackets then url in parenthesis</p>
<p>However the user clicking the link kills the Github page and it opens a new page, which is a really bad web UX…</p>
<p>Anyone has an idea to fix this?</p>
<p>Thanks you!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>If this is the same issue as here:</p>
<aside class="quote quote-modified" data-post="1" data-topic="2326">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/cant-get-links-to-open-in-new-tab-bis/2326">Can't get links to open in new tab (bis)</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    Sorry to create a new post about this but the issue seems very old and was posted in many places (eg. <a href="https://github.com/mojombo/github-flavored-markdown/issues/28" rel="nofollow noopener">https://github.com/mojombo/github-flavored-markdown/issues/28</a>) and I could not find a way to make target="_blank" work in a html link yet. 
I do not like that users leave the github page when clicking such link and would prefer that they get a new tab to see the result. 
My example would be like this: 
blah blah &lt;a href="http://htmlpreview.github.com/?https://github.com/&lt;repo&gt;/&lt;path&gt;/blob/master…
  </blockquote>
</aside>

<p>I’m afraid the answer remains the same. At present there is no way to force a link in a markdown file on GitHub to open in a new tab/window. The person clicking will have to decide that for themselves.</p>

**Comment by tlsLizard** [comment]
<p>Thank you canuckjacq for your feedback.<br>
I believe this would be a good improvement for the future. This type of UX is very bad for the website itself, since the user suddenly leaves it  (I got this opinion not by myself, but by senior web designers)<br>
In the meantime I will circumvent the pb by simply offering the user to copy/paste the URL as a string in their browser.<br>
Cheers</p>
